.github/workflows/deploy_infrastructure.yaml

```diff
@@ -19,16 +19,15 @@ jobs:
         role-to-assume: ${{ vars.ROLE_ARN }}
         aws-region: us-east-1
     - name: Install dependencies
-      run: npm ci --legacy-peer-deps
+      run: npm ci
     - name: Deploy Dev
       env:
         CDK_DEFAULT_ACCOUNT: ${{ vars.CDK_DEFAULT_ACCOUNT }}
         CDK_DEFAULT_REGION: ${{ vars.CDK_DEFAULT_REGION }}
         EMAIL_SENDER: ${{ vars.EMAIL_SENDER }}
         ADMIN_EMAIL: ${{ vars.ADMIN_EMAIL }}
         BASE_DOMAIN: ${{ vars.BASE_DOMAIN }}
-        DEV_MONGO_MEMORY: ${{ vars.DEV_MONGO_MEMORY || '2048' }}
-        DEV_MONGO_CPU: ${{ vars.DEV_MONGO_CPU || '1024' }}
-        DEV_APP_MEMORY: ${{ vars.DEV_APP_MEMORY || '2048' }}
-        DEV_APP_CPU: ${{ vars.DEV_APP_CPU || '1024' }}
+        APP_MEMORY: ${{ vars.APP_MEMORY || '2048' }}
+        APP_CPU: ${{ vars.APP_CPU || '1024' }}
+        DPDASH_DEV: "1"
```

Comment:
should we use a more descriptive env var name? This seems fairly in name generic for a specific guard.

Reply:
04f617b